eos-tech-support/eos-download-image

```diff
@@ -284,6 +284,7 @@ class EosDownloadImage(object):
         dest = os.path.join(self.args.outdir, filename)
         log("Fetching Windows Installer")
         self.download(url, dest)
+        print(dest)
```

Comment:
I've gone a little belt-and-braces, and documented it both in docstrings, and in `--help`. Happy to remove one or the other if you prefer, but now that you mention it, it does seem worth mentioning in `--help` as well, since it's part of the "API" of this tool.